lib/wallet/node.js

```diff
@@ -108,6 +108,12 @@ class WalletNode extends Node {
     await this.http.open();
     await this.handleOpen();
 
+    const clientInfo = await this.client.getInfo();
+    this.chain = {
```

Comment:
I think we should a different var than `this.chain` for this.

Reply:
Making this change in `walletdb.js` and `plugin.js`